### ISSUE TITLE
bug: Failed to verify your browser - Vercel

### ISSUE THREAD

**Issue by EwertonRicardo** [ionic-team/ionic-docs#3880]
### Describe the Bug

I've got the error Failed to verify your browser when trying to access the ionic documentation.

I wonder if this error only occurs to me but some friends also do have the same problem on accessing the documentation.

### Expected Behavior

Access the documentation with success

### Steps to Reproduce

Just try to access it: https://ionicframework.com/docs/api/loading

### Screenshots

![image](https://github.com/user-attachments/assets/88c98d8f-9f66-4a56-bd1e-8bbd62baed01)

### Operating System

MAC

### Browser

Chrome

**Comment by ShonGee** [comment]
For the moment it seems like you can clear your site settings Chrome, but it's super frustrating and each check is ~5 seconds per page load. Also all of the code examples aren't currently loading.

**Comment by EwertonRicardo** [comment]
> For the moment it seems like you can clear your site settings Chrome, but it's super frustrating and each check is ~5 seconds per page load. Also all of the code examples aren't currently loading.

Yeah, it's frustrating. Sometimes I can access the documentation but the examples do not load and if I keep reloading the page, the CSS is gone, crazy.
